app/cdap/components/ConnectionList/Components/ConnectionTabs/index.tsx

```diff
@@ -101,29 +101,21 @@ export default function ConnectionsTabs({
                   }
                 }}
                 label={
-                  index > 1 ? (
-                    connectorType.canBrowse ? (
-                      <TabLabelCanBrowse
-                        count={undefined}
-                        label={connectorType.name}
-                        index={index}
-                      />
-                    ) : (
-                      <TabLabelCanSample
-                        label={connectorType.name}
-                        entity={connectorType}
-                        initialConnectionId={connectionIdProp}
-                        toggleLoader={props.toggleLoader}
-                        setIsErrorOnNoWorkSpace={setIsErrorOnNoWorkSpace}
-                      />
-                    )
-                  ) : (
+                  [0, 1].includes(index) || connectorType.canBrowse ? (
                     <TabLabelCanBrowse
                       label={connectorType.name}
                       count={index === 0 ? connectorType.count : undefined}
                       index={index}
                       icon={connectorType.icon}
                     />
+                  ) : (
+                    <TabLabelCanSample
+                      label={connectorType.name}
+                      entity={connectorType}
+                      initialConnectionId={connectionIdProp}
+                      toggleLoader={props.toggleLoader}
```

Comment:
expand props in component args list and don't use props. here

Reply:
Done